SwiftGRPC.podspec

```diff
@@ -41,6 +41,6 @@ Pod::Spec.new do |s|
   s.source_files = 'Sources/SwiftGRPC/*.swift', 'Sources/SwiftGRPC/**/*.swift', 'Sources/CgRPC/shim/*.[ch]'
   s.public_header_files = 'Sources/CgRPC/shim/cgrpc.h'
 
-  s.dependency 'gRPC-Core', '~> 1.23.0'
+  s.dependency 'gRPC-Core', '~> 1.24.1'
```

Comment:
Should this be 1.24.3?

Reply:
Sorry, I mistook .1 for .3. The .1 has already been pushed; should we keep that (given that `pod update` would install the .3, anyway), or push an 0.11.1 version?

Reply:
I think it's probably fine